evap/staff/views.py

```diff
@@ -1187,8 +1191,14 @@ def user_import(request):
 @staff_required
 def user_edit(request, user_id):
     @receiver(RewardPointGranting.granted_by_removal)
-    def notify_reward_points(users, **_kwargs):
-        messages.info(request, _('The removal of courses has granted the user "{}" reward points for the active semester.'.format(users[0].username)))
+    def notify_reward_points(granted, **kwargs):
+        messages.info(request,
```

Comment:
`_kwargs`, please, since it's unused (that changed in a PR merged today, maybe it got lost during the merge?).

also, could you `assert len(granted) == 1`? because it's not immediately obvious that this is assumption you're making is always true here.

also, if you call the elements of `granted` `transaction`, then maybe the list should be called `transactions`. although, maybe `granting` and `grantings` would be better, to not introduce any new names unnecessarily? especially if you choose to use the Granting objects instead of the tuples/dictionaries.

Reply:
Yes, I wasn't careful enough while rebasing stuff